draft-ietf-scitt-architecture.md

```diff
@@ -581,9 +577,7 @@ To register a Signed Statement, the Transparency Service performs the following
 
 1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements on behalf of one or more issuers.
 Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
-1. **Issuer Verification:** The Transparency Service MUST perform resolution of the Issuer's identity, which may be different than the Client identity.
```

Comment:
Would you be ok to have this clarified that the verification is done before issuing a receipt?

Reply:
Yep, but I'd like to adopt the more extreme position and find out if my sentiments stand up to the experience of the audience!